packages/web/components/complex/paginated-table.tsx

```diff
@@ -156,31 +164,29 @@ export const PaginatedTable = ({
         {virtualRows.map((virtualRow) => {
           const row = rows[virtualRow.index] as Row<ObservablePoolWithMetric>;
           return (
-            <tr
-              key={row.id}
-              className="transition-colors focus-within:bg-osmoverse-700 focus-within:outline-none hover:cursor-pointer hover:bg-osmoverse-800"
-              onClick={() => {
-                router.push(getPoolLink(row.original.queryPool));
-              }}
-            >
-              {row.getVisibleCells().map((cell) => {
-                return (
-                  <td key={cell.id}>
-                    <Link
-                      href={getPoolLink(row.original.queryPool)}
-                      key={virtualRow.index}
-                      passHref
-                      onClick={(e) => e.stopPropagation()}
-                    >
-                      {flexRender(
-                        cell.column.columnDef.cell,
-                        cell.getContext()
-                      )}
-                    </Link>
-                  </td>
-                );
-              })}
-            </tr>
+          <tr
+          key={row.id}
+          className="transition-colors focus-within:bg-osmoverse-700 focus-within:outline-none hover:cursor-pointer hover:bg-osmoverse-800"
+          onClick={(e) => {
```

Comment:
Again I don't like the use of the onClick event to handle the links, we lose the prefetch of the next links.
I suggest you look at how we implement this functionality in the asset table to use it as a reference:
https://github.com/osmosis-labs/osmosis-frontend/blob/stage/packages/web/components/table/index.tsx#L184